site/docs/spec.md

```diff
@@ -206,19 +206,22 @@ The schema of a manifest file is a struct called `manifest_entry` with the follo
 | **`104  file_size_in_bytes`**     | `long`                                | Total file size in bytes                                                                                                                                                                             |
 | ~~**`105 block_size_in_bytes`**~~ | `long`                                | **Deprecated. Always write a default value and do not read.**                                                                                                                                        |
 | **`106  file_ordinal`**           | `optional int`                        | Ordinal of the file w.r.t files with the same partition tuple and snapshot id                                                                                                                        |
-| **`107  sort_columns`**           | `optional list`                       | Columns the file is sorted by                                                                                                                                                                        |
+| **`107  sort_columns`**           | `optional list`                       | Columns the file is sorted by [2]. If a column has type `float` or `double` and contains `NaN`, it must not be in `sort_columns`.                                                                    |
```

Comment:
Can a column containing nulls be in `sort_columns`? If so, are the nulls at the beginning, the end, either, or arbitrarily interspersed?

Reply:
Sort columns is currently not used and we intend to remove it. It sounded like a good idea at first, but we will need direction and null handling rules. What we are planning to do instead is to define sort orders in table metadata and attach them to files by ID. So don't worry about this, we'll remove it.